src/main/java/org/threadly/concurrent/PrioritySchedulerStatisticTracker.java

```diff
@@ -518,72 +549,6 @@ public int getCurrentRunningCount() {
   }
   
   /**
-   * Call to see how frequently tasks are able to immediately get a thread to execute on (and NOT 
```

Comment:
I will have another pull request where these are marked as deprecated for a 3.7.0 release.